infrastructure/azure.tf

```diff
@@ -16,177 +16,196 @@ provider "null" {
+  type = "string"
 }
 
 variable "message_blob_container" {
-  default = "message-content"
+  default     = "message-content"
   description = "Name of the message container blob"
 }
 
 variable "azurerm_functionapp" {
-  type = "string"
+  type        = "string"
   description = "Name of the main Functions application"
 }
 
 variable "azurerm_functionapp_storage_account" {
-  type = "string"
+  type        = "string"
   description = "Name of the storage account for functions"
 }
 
 variable "azurerm_functionapp_git_repo" {
-  default = "https://github.com/teamdigitale/digital-citizenship-functions"
+  default     = "https://github.com/teamdigitale/digital-citizenship-functions"
   description = "The GitHub repository that must be associated to the function app"
 }
 
 variable "azurerm_functionapp_git_branch" {
-  default = "funcpack-release-latest"
+  default     = "funcpack-release-latest"
   description = "The branch of the GitHub repository that must be associated to the function app"
 }
 
 # Name of the storage queue for email notifications
 variable "azurerm_storage_queue_emailnotifications" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the storage queue for created messages
 variable "azurerm_storage_queue_createdmessages" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the CosmosDB account
 variable "azurerm_cosmosdb" {
-    type = "string"
+  type = "string"
 }
 
 variable "azurerm_cosmosdb_documentdb" {
-  type = "string"
+  type        = "string"
   description = "Name of CosmosDB Database"
 }
 
 variable "azurerm_cosmosdb_collections" {
-  type = "map"
+  type        = "map"
   description = "Name and partition keys of collections that must exist in the CosmosDB database"
 }
 
 # Name of the App Service Plan resource
 variable "azurerm_app_service_plan" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the App Service Plan for developer portal
 variable "azurerm_app_service_plan_portal" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the App Service for developer portal
 variable "azurerm_app_service_portal" {
-    type = "string"
+  type = "string"
 }
 
 variable "app_service_portal_git_repo" {
-  type = "string"
+  type        = "string"
   description = "URL of the GitHub repository providing the source of the App Service Portal"
 }
 
 variable "app_service_portal_git_branch" {
-  default = "master"
+  default     = "master"
   description = "Branch of the GitHub repository providing the source of the App Service Portal"
 }
 
 # Redirect to this page after developer portal login
 variable "app_service_portal_post_login_url" {
-    type = "string"
+  type = "string"
 }
 
 # Redirect to this page after developer portal logout
 variable "app_service_portal_post_logout_url" {
-    type = "string"
+  type = "string"
 }
 
 # Name of the API management resource
 variable "azurerm_apim" {
-    type = "string"
+  type = "string"
+}
+
+variable "azurerm_apim_sku" {
+  type = "string"
```

Comment:
add a description for new variables (eventually we'll set the description for the existing variables too)